public/HikeConfig.txt
Hike Config files. Copy/Paste Config function to config.js . Each hike will have its own config function.

// Smith Rock
const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "lefty",
        hidden: false,
        title: "Smith Rock",
        image:
          "https://cdn.crowdriff.com/in-use/b6d36708-d6b0-cc35-e70d-70adeea5e461/750.jpg",
        description:
          "Smith Rock State Park is a state park located in central Oregon's high desert near the towns of Redmond and Terrebonne...",
        Regulations: [
            "Day use parking permit required",
            "Open dawn to dusk",
            "Information kiosk, restrooms, picnic area",
            "Keep dogs on leash",
            "Share some trails with mountain bikers",
        ],
        location: {
          center: [-121.13789, 44.36515],
          end: [-121.14257, 44.37290]
          zoom: 14.5,
          pitch: 70,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "righty",
        hidden: false,
        title: "Smith Rock",
        image:
          "https://cdn.crowdriff.com/in-use/20db2ce2-5751-bb01-eeeb-98f8719e97fc/750.jpg",
        description:
          "The Misery Ridge Trail in Smith Rock State Park is an exceptional day hike...",
        location: {
          center: [-121.13789, 44.36515],
          zoom: 14.5,
          pitch: 60,
          bearing: -43.2,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "lefty",
        hidden: false,
        title: "Smith Rock",
        image:
          "https://www.oregonhikers.org/w/images/f/fb/SmithRockTrhd1.jpg",
        description:
          "The park is characterized by its 500-foot-tall imposing golden rock spires...",
        location: {
          center: [-121.13789, 44.36515],
          zoom: 13.52,
          pitch: 8.01,
          bearing: 0.0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Smith Rock",
        image:
          "https://www.oregonhikers.org/w/images/thumb/5/5a/Three_Sisters_looming%2C_Burma_Road%2C_Smith_Rock.jpg/800px-Three_Sisters_looming%2C_Burma_Road%2C_Smith_Rock.jpg",
        link:"https://www.outdoorproject.com/united-states/oregon/smith-rock-state-park",
        description:
          "If you enjoy scenic views of deep river canyons or rock climbing, Smith Rock State Park is the place for you...",
        location: {
          center: [-121.13789, 44.36515],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  };

  // Angel's Rest 
  const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "righty",
        hidden: false,
        title: "Angel's Rest",
        image:
          "https://www.oregonhikers.org/w/images/6/6a/GorgeSunsetAngelsRest.jpg",
        description:
          "Angel's Rest is a popular trail in the Columbia River Gorge...",
        Regulations: [
            "Parking can be a problem. Try to get there early (before 8:30 a.m. on weekends), or hike during the week. Park only in designated spots at one of the two parking areas.",
            "Day use only: open 6:00 a.m. - 10:00 p.m.",
            "Closest restrooms at the Bridal Veil Trailhead, less than one mile west",
            "Keep dogs on leash",
            "Best not to hike this trail under icy, windy, or very wet conditions",
        ],
        location: {
          center: [-122.17241, 45.56065],
          end: [-122.15299, 45.56431],
          zoom: 14.5,
          pitch: 80,
          bearing: 120,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "righty",
        hidden: false,
        title: "Angel's Rest",
        image:
          "https://www.oregonhikers.org/w/images/thumb/3/34/Cape_Horn%2C_Angel%27s_Rest_Trail.jpg/500px-Cape_Horn%2C_Angel%27s_Rest_Trail.jpg",
        description:
          "The platy andesite that composes this promontory is part of a million-year-old lava flow from Larch Mountain, a member of the Boring Lava Field. The summit is characterized by a long, rocky spine surrounded on three sides by cliffs, boasting a striking 270 degree view!",
        location: {
          center: [-122.17241, 45.56065],
          zoom: 14.5,
          pitch: 70,
          bearing: 360,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "lefty",
        hidden: false,
        title: "Angel's Rest",
        image:
          "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.pCqeD3pohBQh313teKMwBAHaDt%26pid%3DApi&f=1&ipt=f5c5bb51953dcf36451354a62fee95248b92b0546f071c19be38299377c669ac&ipo=images",
        description:
          "While you can't see any of the Cascade volcanoes from the top, you do get great vantages of Beacon Rock, Silver Star Mountain, and many other landmarks.",
        location: {
          center: [-122.17241, 45.56065],
          zoom: 13.52,
          pitch: 8.01,
          bearing: 0.0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Angel's Rest",
        image:
          "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.CcgT8kL8Y8ju83qZlSsihQHaC3%26pid%3DApi&f=1&ipt=74f7aafbc9556956b43d32fae48ee9e2aed66a481cf433f6b3ad9fbda11b74cb&ipo=images",
        link: "https://www.oregonhikers.org/field_guide/Angel%27s_Rest_Hike",
        description:
          "The real draw, however, is the perspective of the Columbia River below - like you're on a balcony over a great auditorium.",
        location: {
          center: [-122.17241, 45.56065],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  };

// eagle creek to tunnel falls

const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "lefty",
        hidden: false,
        title: "Eagle Creek to Tunnel Falls",
        image:
          "https://www.oregonhikers.org/w/images/thumb/3/35/PunchbowlFalls.jpg/500px-PunchbowlFalls.jpg",
        description:
          "Tunnel Falls is the traditional turnaround spot for many on the Eagle Creek Trail, and it is indeed a splendid climax...",
        Regulations: [
            "Northwest Forest Pass (or America the Beautiful Pass) required; fee kiosk at the trailhead",
            "Restrooms, picnic tables, information kiosk, nearby campground",
            "Limited parking; if lot is full, go back to the Eagle Creek Day Use Trailhead",
            "Keep dogs on leash",
            "Overnight camping permitted above the High Bridge; campfires not allowed!",
        ],
        location: {
          center: [-121.91970, 45.636574],
          end: [-121.85199, 45.58489],
          zoom: 12.5,
          pitch: 50,
          bearing: 130,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "righty",
        hidden: false,
        title: "Eagle Creek to Tunnel Falls",
        image:
          "https://www.oregonhikers.org/w/images/thumb/d/d6/TunnelFalls1.jpg/600px-TunnelFalls1.jpg",
        description:
          "Over the last thousand years, the falls have carved a majestic basin here, plunging 160 feet from towering basalt cliffs to the rushing creek bed below...",
        location: {
          center: [-121.91970, 45.636574],
          zoom: 12.5,
          pitch: 60,
          bearing: -43.2,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "lefty",
        hidden: false,
        title: "Eagle Creek to Tunnel Falls",
        image:
          "https://hikingproject.com/assets/photos/hike/7016924_smallMed_1554828543.jpg?cache=1733552775",
        description:
          "While this scene is impressive enough, the most bewildering aspect of the falls in the passageway behind them. Trail builders from the early 1900s blasted a tunnel behind the falls and then across the sheer rock face on their far side, making this area the slickest and perhaps most precarious section of trail in the Columbia River Gorge.",
        location: {
          center: [-121.91970, 45.636574],
          zoom: 12,
          pitch: 8.01,
          bearing: 0.0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Eagle Creek to Tunnel Falls",
        image:
          "https://hikingproject.com/assets/photos/hike/7011604_smallMed_1554559703.jpg?cache=1733552651",
        link:"https://www.hikingproject.com/trail/7022469/eagle-creek-trail-to-tunnel-falls",
        description:
          "Looking downstream from the falls, you can see where the two forks of Eagle Creek converge, meeting at Grand Union Falls",
        location: {
          center: [-121.91970, 45.636574],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  };

  // Multnomah-Wahkeena Loop
  const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "lefty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:
          "https://www.oregonhikers.org/w/images/d/da/Ecola1.JPG",
        description:
          "Wahkeena Falls Loop is a beautiful experience located in the Columbia River Gorge National Scenic Area that brings hikers to one of the best waterfalls on the Oregon side...",
        Regulations: [
            "For parking at the Multnomah Falls Lodge between Memorial Day and Labor Day weekends, a Timed Use Permit ($2 fee) will be required for each personal vehicle between 9 a.m. and 6 p.m. In addition, as of 2024 there's an additional parking fee ($10 and up) required by the concessionaire (Sasquatch Shuttle) that operates the parking lot here.",
            "For the I-84 Exit 31 parking area, between Memorial Day and Labor Day weekends, you will need to purchase reserved tickets from Recreation.gov. Booking early in the morning or later in the afternoon is best for ensuring a spot.",
            "Limited parking; if lot is full, go back to the Eagle Creek Day Use Trailhead",
            "Keep dogs on leash",
            "Restrooms, restaurant, visitor center at Multnomah Falls Lodge; picnic area, restrooms, information kiosk near Wahkeena Trailhead",
        ],
        location: {
          center: [-122.11841, 45.57893],
          end: [-122.12809, 45.57535],
          zoom: 14,
          pitch: 50,
          bearing: 130,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "lefty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:
          "https://hikingproject.com/assets/photos/hike/7030417_smallMed_1554926948.jpg?cache=1733553192",
        description:
          "Start from the Wahkeena Picnic Area and take Wahkeena which passes by several waterfalls and ascends Wahkeena Creek past the springs which feed the waterfall...",
        location: {
          center: [-122.11841, 45.57893],
          zoom: 13.7,
          pitch: 40,
          bearing: -43.2,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "righty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:
          "https://hikingproject.com/assets/photos/hike/7024271_smallMed_1554846626.jpg?cache=1733552963",
        description:
          "Lots to see and do initially as you climb up Wahkeena Trail. The trail is well maintained and easy to follow. The trail is a bit steep in places but not too bad...",
        location: {
          center: [-122.11841, 45.57893],
          zoom: 14,
          pitch: 8.01,
          bearing: 0.0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:
          "https://hikingproject.com/assets/photos/hike/7016934_smallMed_1554828562.jpg?cache=1733552776",
        link:"https://www.hikingproject.com/trail/7085296/wahkeena-falls-devils-rest-angels-rest-loop",
        description:
          "Looking downstream from the falls, you can see where the two forks of Eagle Creek converge, meeting at Grand Union Falls",
        location: {
          center: [-122.11841, 45.57893],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  }

// Mirror Lake

const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "lefty",
        hidden: false,
        title: "Mirror Lake",
        image:
          "https://www.oregonhikers.org/w/images/e/e8/MirrorLakeAlpenglow.jpg",
        description:
          "The Mirror Lake Trail has the misfortune of being one of the few hikes accessible directly from Highway 26 and thus suffers a crush of visitors on weekends. The attention is well-deserved: The well-graded trail takes you to a serene mountain lake with picture-postcard views of Mount Hood and rugged Tom Dick and Harry Mountain towering above. With the old trailhead decommissioned in 2018, visitors now have a longer hike but with somewhat less elevation gain from the new Mirror Lake Trailhead. The new trail was machine-built, so it's wide and verged with debris from the operation. There are 10 footbridges on this new section; nine of them were airlifted in by helicopter.",
        Regulations: [
            "Northwest Forest Pass required",
            "Day use only",
            "Restrooms, picnic table, information kiosk at trailhead",
            "Keep dogs on leash",
        ],
        location: {
          center: [-121.77728, 45.30277],
          end: [-121.79286, 45.29695],
          zoom: 14.5,
          pitch: 70,
          bearing: 130,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "lefty",
        hidden: false,
        title: "Mirror Lake",
        image:
          "https://www.oregonhikers.org/w/images/thumb/3/30/MirrorLake.jpg/500px-MirrorLake.jpg",
        description:
          "The Mirror Lake Trail has the misfortune of being one of the few hikes accessible directly from Highway 26 and thus suffers a crush of visitors on weekends. The attention is well-deserved: The well-graded trail takes you to a serene mountain lake with picture-postcard views of Mount Hood and rugged Tom Dick and Harry Mountain towering above. With the old trailhead decommissioned in 2018, visitors now have a longer hike but with somewhat less elevation gain from the new Mirror Lake Trailhead. The new trail was machine-built, so it's wide and verged with debris from the operation. There are 10 footbridges on this new section; nine of them were airlifted in by helicopter.",
        location: {
          center: [-121.77728, 45.30277],
          zoom: 13.7,
          pitch: 70,
          bearing: -43.2,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "righty",
        hidden: false,
        title: "Mirror Lake",
        image:
          "https://www.oregonhikers.org/w/images/thumb/8/85/Mt._Hood_from_Mirror_Lake.jpg/500px-Mt._Hood_from_Mirror_Lake.jpg",
        description:
          "From the plaza at the parking area, find the paved ADA trail that leads down a slope behind the restrooms. Six winding switchbacks descend 0.2 miles to a wide footbridge over Camp Creek, where the paved section ends. You're in a secondary forest of Douglas-fir, western hemlock, silver fir, noble fir, and western red-cedar. Before Camp Creek, a pole-and-rail barrier prevents shortcutting at a switchback.After Camp Creek, you're on a wide dirt trail that often has protruding roots and embedded rocks. Cross two footbridges over seasonally dry gullies. The next bridge crosses the creek that drains the Comerford Lakes. Then you'll rise gradually to cross more footbridges over trickling creeks that feed small bogs below. Rhododendrons enter the understory, and you'll see springboard notches on rotting stumps. The trail undulates along, passing over more bridges, and soon crosses Mirror Lake Creek. After crossing an old logging road, you'll bend sharp left to rise steeply and intersect the old tread which came up from the U.S. 26 trailhead. Whole trees have been felled over this path to decommission it.Switchback on the narrower old tread, and pass a vine maple-verged patch of talus. Walk on another old logging road for a few yards, pass through a devil's club thicket, and switchback up twice to where a new pole-and-rail fence prevents cutting corners. Reach the Mirror Lake-Mirror Lake Shore Trail North Junction, and go left to begin your clockwise tour around Mirror Lake.",
        location: {
          center: [-121.77728, 45.30277],
          zoom: 14,
          pitch: 10,
          bearing: 0.0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Mirror Lake",
        image:
          "https://live.staticflickr.com/4882/43974838530_8023ab3489_k.jpg",
        link:"https://www.oregonhikers.org/field_guide/Mirror_Lake_Hike",
        description:
          "Hikers visiting Mirror Lake will enjoy spectacularly close and personal views of Mount Hood and Tom Dick and Harry Mountain.  Visit in mid-July and you'll also enjoy the trail's outstanding abundance of Pacific rhododendron in bloom, as well as Douglas's spiraea surrounding the lake's marshy southern end.",
        location: {
          center: [-122.11841, 45.57893],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  };

export default config;

// Cascade Head 

const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "righty",
        hidden: false,
        title: "Cascade Head",
        image:
          "https://www.oregonhikers.org/w/images/f/f4/CascadeHead3.JPG",
        description:
          "This is one of the truly amazing places on the Oregon Coast. It has spectacular views down to Three Rocks, the Salmon River Estuary, and the headlands north of Lincoln City. The grassy headland is also home to an endangered species of butterfly, the Oregon silverspot, and at least two rare plant species.",
        Regulations: [
            "Dogs are not permitted on this trail",
            "Off-trail travel is not permitted",
            "Last 0.6 miles of trail as well as the upper Nature Conservancy Trailhead closed from January 1st to July 15th.",
            "Restrooms at Knights Park Trailhead",
        ],
        location: {
          center: [-123.99234, 45.04183],
          end: [-123.98842, 45.06068],
          zoom: 14.5,
          pitch: 70,
          bearing: 30,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "righty",
        hidden: false,
        title: "Cascade Head",
        image:
          "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse4.mm.bing.net%2Fth%3Fid%3DOIP.DlZMfYEnCndoDvibmSpgwgHaDy%26pid%3DApi&f=1&ipt=48f93e1bb3f6aa134768c7cc2daae98e018be8e169cab5ac8ed7d15d913bc7ff&ipo=images",
        description:
          "The hike begins at Knights Park. The first part of trail follows a path of wood shavings scant feet from Three Rocks Road. Cross the road and you'll find the path headed north on the shoulder of Savage Road. The trail dips from the road to reach a wooden bridge, and then parallels the road through the trees. At the crest of a hill, the path crosses the road and follows its west shoulder. A sign at the bottom of the hill marks the original trailhead, where the trail leaves the road for good.",
        location: {
          center: [-123.99234, 45.04183],
          zoom: 14,
          pitch: 70,
          bearing: -43.2,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "righty",
        hidden: false,
        title: "Cascade Head",
        image:
          "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.RpOy7N0Q6dXouW0T0Ie8SAHaEK%26pid%3DApi&f=1&ipt=adbdd715948c2c654ff129b52a16b895d99a501eb5ddeb37402b515599e5b45c&ipo=images",
        description:
          "This headland meadow is a very rare ecosystem. There are two rare plants that call this hill home, the Cascade Head catchfly (Silene douglasii var oraria) and the hairy-stemmed checkermallow (Sidalcea hirtipes). Both of these species make their strongest stand here. It's estimated the 99% of all Cascade Head catchflies live on this one hill.",
        location: {
          center: [-123.99234, 45.04183],
          zoom: 14,
          pitch: 10,
          bearing: 60,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Cascade Head",
        image:
          "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.2O-hriXSCC3bhoSEra-RwAHaFj%26pid%3DApi&f=1&ipt=ba00787b20494d904015158126680bd839daca06045a2d1afb36742faec697fa&ipo=images",
        link:"https://www.oregonhikers.org/field_guide/Cascade_Head_Hike",
        description:
          "With the roadside hike done, you'll begin climbing steeply, sometimes on stairs, through an old growth coastal forest. Douglas-fir and Sitka spruce dominate the area. You're on National Forest land in this short section, but please stay on the trail as private property is close. You'll climb about 270 feet in the first 2/10 of a mile to a grove of Sitka spruce indicated by a 13 marker. The climb eases here.",
        location: {
          center: [-123.99234, 45.04183],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  };

export default config;

// Multnomah-Wahkeena Loop

const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "lefty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:
          "https://www.oregonhikers.org/w/images/d/da/Ecola1.JPG",
        description:
          "This popular loop is a waterfall lover's paradise. There are six officially named waterfalls on this trip as well as countless cascades and intermittent falls. The loop can be hiked either direction, and it can be started from either trailhead.",
        Regulations: [
            "Restrooms, restaurant, visitor center at Multnomah Falls Lodge; picnic area, restrooms, information kiosk near Wahkeena Trailhead",
            "Parking Fee",
            "Dogs on leash"
        ],
        location: {
          center: [-122.11841, 45.57893],
          end: [-122.12809, 45.57535],
          zoom: 14.5,
          pitch: 70,
          bearing: 180,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "righty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:
          "https://images.alltrails.com/eyJidWNrZXQiOiJhc3NldHMuYWxsdHJhaWxzLmNvbSIsImtleSI6InVwbG9hZHMvcGhvdG8vaW1hZ2UvOTA0OTM5MjYvMjgyOTk1MTA3NmZkYTFjZGZkNTYwYWE3ODM1ZDkxMjYuanBnIiwiZWRpdHMiOnsidG9Gb3JtYXQiOiJqcGVnIiwicmVzaXplIjp7IndpZHRoIjo1MDAsImhlaWdodCI6NTAwLCJmaXQiOiJpbnNpZGUifSwicm90YXRlIjpudWxsLCJqcGVnIjp7InRyZWxsaXNRdWFudGlzYXRpb24iOnRydWUsIm92ZXJzaG9vdERlcmluZ2luZyI6dHJ1ZSwib3B0aW1pc2VTY2FucyI6dHJ1ZSwicXVhbnRpc2F0aW9uVGFibGUiOjN9fX0=",
        description:
          "The trail begins at Multnomah Falls Lodge, a historic building constructed to serve early automobile travelers in 1925. From a photographer's viewpoint, get a head on vista of both the lower and main tiers of Multnomah Falls and the picturesque span of the Benson Bridge.",
        location: {
          center: [-122.11841, 45.57893],
          zoom: 14,
          pitch: 70,
          bearing: 360,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "righty",
        hidden: false,
        title: "Multnomah-Wahkeena Loop",
        image:

src/config.js
const config = {
    style: "mapbox://styles/mapbox/satellite-streets-v12", // Supports 3D terrain
    accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
    showMarkers: true,
    markerColor: "rgb(0,127,92)",
    theme: "dark",
    use3dTerrain: true,
    auto: false,
    title: "",
    subtitle: "",
    byline: "Scroll Down to Explore",
    footer:
      '<br> Created By <a href="https://interactive-reality.netlify.app/" target="_blank">Richard Brown</a>',
    chapters: [
      {
        id: "slug-style-id",
        alignment: "lefty",
        hidden: false,
        title: "Tamolitch Blue Pool",
        image:
          "https://www.oregonhikers.org/w/images/5/51/Tamolitch_Falls_and_Blue_Pool%2C_McKenzie_River_Trail.jpg",
        description:
          "Head out on this 3.9-mile out-and-back trail near Blue River, Oregon. Generally considered a moderately challenging route, it takes an average of 1 h 28 min to complete. This is a very popular area for birding, hiking, and mountain biking, so you'll likely encounter other people while exploring. The trail is open year-round and is beautiful to visit anytime. Dogs are welcome, but must be on a leash.",
        Regulations: [
            "Restrooms, information kiosk, nearby campground",
            "Share trail with mountain bikers"
        ],
        location: {
          center: [-122.03567, 44.29022],
          end: [-122.0269, 44.3120],
          zoom: 13.5,
          pitch: 70,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "second-identifier",
        alignment: "lefty",
        hidden: false,
        title: "Tamolitch Blue Pool",
        image:
          "https://images.alltrails.com/eyJidWNrZXQiOiJhc3NldHMuYWxsdHJhaWxzLmNvbSIsImtleSI6InVwbG9hZHMvcGhvdG8vaW1hZ2UvNDc1ODgxNzUvZDQ3ODkzOWVhOGEzNzEyNzk3MTUwZjI3ZDQ2YmFlMzMuanBnIiwiZWRpdHMiOnsidG9Gb3JtYXQiOiJqcGVnIiwicmVzaXplIjp7IndpZHRoIjo1MDAsImhlaWdodCI6NTAwLCJmaXQiOiJpbnNpZGUifSwicm90YXRlIjpudWxsLCJqcGVnIjp7InRyZWxsaXNRdWFudGlzYXRpb24iOnRydWUsIm92ZXJzaG9vdERlcmluZ2luZyI6dHJ1ZSwib3B0aW1pc2VTY2FucyI6dHJ1ZSwicXVhbnRpc2F0aW9uVGFibGUiOjN9fX0=",
        description:
          "The beginning of this easy hike is through an old-growth Douglas fir forest. As the McKenzie River falls away, you walk along a cliff and descend into a dry river bed which eventually comes to an old lava flow.",
        location: {
          center: [-122.03567, 44.29022],
          zoom: 13,
          pitch: 70,
          bearing: 220,
        },
        mapAnimation: "flyTo",
        rotateAnimation: true,
      },
      {
        id: "third-identifier",
        alignment: "righty",
        hidden: false,
        title: "Tamolitch Blue Pool",
        image:
          "https://images.alltrails.com/eyJidWNrZXQiOiJhc3NldHMuYWxsdHJhaWxzLmNvbSIsImtleSI6InVwbG9hZHMvcGhvdG8vaW1hZ2UvOTEyNzg0MzYvN2UxNDJkMzExMDIyMDNjNTEwODVjOTYwMDc3OWRkMGUuanBnIiwiZWRpdHMiOnsidG9Gb3JtYXQiOiJqcGVnIiwicmVzaXplIjp7IndpZHRoIjo1MDAsImhlaWdodCI6NTAwLCJmaXQiOiJpbnNpZGUifSwicm90YXRlIjpudWxsLCJqcGVnIjp7InRyZWxsaXNRdWFudGlzYXRpb24iOnRydWUsIm92ZXJzaG9vdERlcmluZ2luZyI6dHJ1ZSwib3B0aW1pc2VTY2FucyI6dHJ1ZSwicXVhbnRpc2F0aW9uVGFibGUiOjN9fX0=",
        description:
          "The trail is not too strenuous but there is some elevation gain as can be seen on the tracks and footing can be tricky in a few places. Be advised the trail passes one ledge with a steep drop-off to the river a short distance before Blue Pool.",
        location: {
          center: [-122.03567, 44.29022],
          zoom: 14,
          pitch: 10,
          bearing: 60,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
      {
        id: "fourth-chapter",
        alignment: "lefty",
        hidden: false,
        title: "Tamolitch Blue Pool",
        image:
          "https://www.oregonhikers.org/w/images/1/13/North_end_of_the_Blue_Pool%2C_McKenzie_River_Trail.jpg",
        link:"https://www.oregonhikers.org/field_guide/Tamolitch_Pool_Hike",
        description:
          "This pool is actually where the McKenzie River emerges after submerging into a lava tube at the Carmen Reservoir, creating the Tamolitch Falls. It's known as a “dry fall” because during especially wet rainy winters the river will flood over the lava bed and waterfall into Tamolitch Pool. ",
        location: {
          center: [-122.03567, 44.29022],
          zoom: 4,
          pitch: 0,
          bearing: 0,
        },
        mapAnimation: "flyTo",
        rotateAnimation: false,
      },
    ],
  };

export default config;